src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -60,11 +60,24 @@ namespace {
 
 enum class test_status
 {
-  passed,
-  warning,
-  failed
+  passed = 0,
+  warning = 1,
+  failed = 2,
+  skipped = 3
 };
 
+static std::string 
+get_test_status(test_status status)
+{
+  switch (status) {
+    case test_status::passed: return "passed"; break;
+    case test_status::warning: return "warning"; break;
+    case test_status::failed: return "failed"; break;
+    case test_status::skipped: return "skipped"; break;
+    case default: return ""; break;
```

Comment:
Don't combine statements to a single line.  Insert line break before the return statement, and remove all the breaks; they don't do anything.  

Alternatively, and since this function is coupled with the enum anyway,  use a 
`static constexpr std::array<const char*, test_status::last> values = { "passed", "...", ...};`
and just` return values[static_cast<int>(status)];`

Reply:
Agreed. I fixed the lines. But I left the switch statement. I think it looks nicer than the std::array.

Reply:
I ended up have to change back to using strings. I had a compilation issue since the ptree requires strings and it got ugly resolving the issue :(. Though rest assured the new variable constants are lower case